test/pages/print_labels/control_plates.spec.js

```diff
@@ -80,61 +73,6 @@ describe('print control plate labels', () => {
     ])
   })
 
-  // TODO: GPL-828 - These tests are duplicated so will be removed once refactored. Need to get it to pass code coverage.
-  describe('setting the status', () => {
-    let vm
-
-    it('default should be idle', () => {
-      vm = mount(ControlPlates, { localVue }).vm
-      expect(vm.isIdle).toBeTruthy()
-    })
-
-    it('when success', () => {
-      wrapper = mount(ControlPlates, {
-        localVue,
-        data() {
-          return {
-            status: statuses.Success,
-            alertMessage: 'I am a success',
-          }
-        },
-      })
-      vm = wrapper.vm
-      expect(vm.isSuccess).toBeTruthy()
-      expect(wrapper.find('.alert').text()).toMatch('I am a success')
-    })
-
-    it('when error', () => {
-      wrapper = mount(ControlPlates, {
-        localVue,
-        data() {
-          return {
-            status: statuses.Error,
-            alertMessage: 'I am a failure',
-          }
-        },
-      })
-      vm = wrapper.vm
-      expect(vm.isError).toBeTruthy()
-      expect(wrapper.find('.alert').text()).toMatch('I am a failure')
-    })
-
-    it('when busy', () => {
-      wrapper = mount(ControlPlates, {
-        localVue,
-        data() {
-          return {
-            status: statuses.Busy,
-            alertMessage: 'I am busy',
-          }
-        },
-      })
-      vm = wrapper.vm
-      expect(vm.isBusy).toBeTruthy()
```

Comment:
This file must have had the status Busy being set somewhere 🤔

Reply:
Well this test is setting that status directly as part of the setup.  I think you were right that the actual Vue page itself never set it that way.  I've fixed that now.